tools/bioimaging/bioimage_inference.xml

```diff
@@ -0,0 +1,79 @@
+<tool id="bioimage_inference" name="Process image using a BioImage.IO model" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
+    <description>with PyTorch</description>
+    <macros>
+        <token name="@TOOL_VERSION@">2.3.1</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <creator>
+        <organization name="BioImage.IO" url="https://bioimage.io/#/" email="" />
+        <person name="Beatriz Serrano-Solano" email="" />
+        <person name="Leonid Kostrykin" email="leonid.kostrykin@bioquant.uni-heidelberg.de" />
+    </creator>
+    <edam_operations>
+        <edam_operation>operation_3443</edam_operation>
+    </edam_operations>
+    <xrefs>
+        <xref type="bio.tools">pytorch</xref>
+        <xref type="biii">pytorch</xref>
+    </xrefs>
+    <requirements>
+        <requirement type="package" version="3.9.12">python</requirement>
+        <requirement type="package" version="2.3.1">pytorch</requirement>
```

Comment:
```suggestion
        <requirement type="package" version="@TOOL_VERSION@">pytorch</requirement>
```

was that the intention behind the version number?

Reply:
My bad, I actually forgot to add this in my review.

Reply:
I think tool's version should not replace Pytorch's version. Something else changes in the tool (making the tool's version to change) while Pytorch remains that same - could be problematic in future. Since, this is the first tool, I would keep the tool's version as 0.0.1

Reply:
This wrapper is rather thin and is mostly "just" a wrapper for `torch.load` (the rest is pre- and post-processing). This suggests that the wrapper should have the same version as the wrapped tool, PyTorch in this case. If the wrapper changes in the future, you would usually increment `@VERSION_SUFFIX`. This is also what I had suggested in https://github.com/bgruening/galaxytools/pull/1391#discussion_r1699899153 and it is more in line with IUC recommendations https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions

Reply:
sorry, I misinterpreted the previous comment. It's restored based on your comment. Thanks!!